docs/UserGuide.adoc

```diff
@@ -295,6 +295,56 @@ To end step-by-step instruction of cook-mode. +
 Format: +
 `end`
 
+=== Health-planner Command
+
+==== Add plan: `add`
+
+Adds a health plan under the user.
```

Comment:
Please leave a new line after each instruction. 
e.g. :
Adds a health plan under the user. +